src/screens/videos/youtube-video.js

```diff
@@ -5,15 +5,15 @@ export function YoutubeVideo({ videoId, onReady, className }) {
     <YouTube
       videoId={videoId}
       className={className}
-      iframeClassName="yt-iframe relative z-10 w-full h-full rounded-t-xl"
+      iframeClassName="yt-iframe relative z-30 w-full h-full rounded-t-xl"
```

Comment:
This z-index also doesn't seem necessary?

Reply:
If I don't put one (even after the other changes), the snow is visible on top of the video. Maybe there's some way to reorder the markup, but it's probably not too big a deal to state it explicitly either.